dependencies.gradle

```diff
@@ -42,6 +43,7 @@ ext {
             ktxFragment         : "androidx.fragment:fragment-ktx:$ktxFragmentVersion",
             viewPager2          : "androidx.viewpager2:viewpager2:$viewPager2Version",
             locaBroadcastManager: "androidx.localbroadcastmanager:localbroadcastmanager:$localBroadcastManagerVersion",
+            lifecycleExtensions : "androidx.lifecycle:lifecycle-extensions:$lifecycleExtensionsVersion"
```

Comment:
Is the lifecycle-extensions library used anywhere?

Reply:
Yes, it's used by the sample app because AppCompatActivity needs to know about view model providers that are defined inside architecture component libraries (like lifecycle extensions)

Reply:
Alright, thank you!